plugin/import_cost.vim

```diff
@@ -35,6 +35,8 @@ let s:default_settings = {
   \ 'split_size': 50,
   \ 'split_pos': 'left',
   \ 'disable_async': 0,
+  \ 'disable_auto': 0,
```

Comment:
Naming of `g:import_cost_disable_auto` is inconsistent with documentation. I think the name should be `g:import_cost_virtualtext` and will be set to `1` by default.